terraform/vars/terraform-dev.tfvars

```diff
@@ -5,8 +5,10 @@ desired_count             = 1
 auto_scaling_min_capacity = 1
 auto_scaling_max_capacity = 5
 
-auth_url                  = "https://api.resourcewatch.org"
-forms_api_url             = "https://api.resourcewatch.org/v1"
+auth_url         = "https://staging-api.resourcewatch.org"
+forms_api_url    = "https://staging-api.resourcewatch.org/v1"
+alerts_api_url   = "https://staging-data-api.globalforestwatch.org/"
```

Comment:
Need to remove the ending `/` to avoid `//` issues with API calls
`"https://staging-data-api.globalforestwatch.org"`